server/notification-providers/notification-provider.js

```diff
@@ -115,6 +116,22 @@ class NotificationProvider {
 
         throw new Error(msg);
     }
+
+    /**
+     * Returns axios config with proxy agent if proxy env is set.
+     * @param {object} axiosConfig - Axios config containing params
+     * @returns {object} Axios config
+     */
+    getAxiosConfigWithProxy(axiosConfig = {}) {
+        if (process.env.NOTIFICATION_PROXY === "true") {
+            const proxyEnv = process.env.https_proxy || process.env.HTTPS_PROXY || process.env.http_proxy || process.env.HTTP_PROXY;
```

Comment:
I was more thinking about something like this.
What do you think?

```suggestion
        const proxyEnv = process.env.notification_proxy || process.env.NOTIFICATION_PROXY;
        if (proxyEnv) {
```